docs/usermanual.md

```diff
@@ -316,29 +429,29 @@ attribute IDs `h` and `t`, then humidity measures are reported this way:
 $ mosquitto_pub -t /json/ABCDEF/id_sen1/attrs/h -m 70 -h <mosquitto_broker> -p <mosquitto_port> -u <user> -P <password>
 ```
 
-In the single measure case, when the published data is not a valid JSON, it is interpreted as binary content. For instance,
-if the following is published to `/json/ABCDEF/id_sen1/attrs/attrHex` topic:
+In the single measure case, when the published data is not a valid JSON, it is interpreted as binary content. For
+instance, if the following is published to `/json/ABCDEF/id_sen1/attrs/attrHex` topic:
 
 ```
 hello
 ```
 
 then the resulting attribute sent to ContextBroker:
 
-{
-   ...
-   "attrHex": {
-     "value": "68656c6c6f"
-     "type": "<the one used at provisiong time for attrHex attribute>"
-   }
-}
+{ ... "attrHex": { "value": "68656c6c6f" "type": "<the one used at provisiong time for attrHex attribute>" } }
```

Comment:
````suggestion
```
{
   ...
   "attrHex": {
     "value": "68656c6c6f"
     "type": "<the one used at provisiong time for attrHex attribute>"
   }
}
```
````

Reply:
Missing trailing comma in L445 in suggestion.

Reply:
L101 should be adapted in a similar way.

Reply:
> Missing trailing comma in L445 in suggestion.

Fixed in original comment